WordPress/src/main/AndroidManifest.xml

```diff
@@ -205,6 +205,7 @@
             android:windowSoftInputMode="stateVisible" />
         <activity
             android:name=".ui.posts.EditPostActivity"
+            android:configChanges="fontScale"
```

Comment:
I'm quite concerned when we touch the EditPostActivity at a very high level since it means we're effectively touching Aztec-standalone too and the scope if very big there.

In this case, I tried out changing the OS font size on a non-blocks post, in Aztec, and it seems that something goes wrong and all the images/videos/paragraphs are one on top of the other:

![aztec-problem-if-configchange-fontsize](https://user-images.githubusercontent.com/1032913/56073152-5b59a000-5da8-11e9-9316-86505e1a4371.gif)

If our best solution at the moment is to adopt the `fontScale` config change and handle it ourselves, I might feel inclined to propose a solution like the option I mentioned in https://github.com/wordpress-mobile/WordPress-Android/pull/8874#issuecomment-455613694, where the idea is to introduce a subclass of the EditPostActivity only to be able to specialize its manifest definition and similar.

In the meantime, can you check the issue shown in the gif @daniloercoli to see if it's fixable? Thanks!

Reply:
Yes, the issue I think is fixable without restarting Aztec but requires small changes to it. Calling `RefreshText` alone doesn't fix it, instead we need to reload the pictures/videos and re-layout them on configChanges. It alone is a rather small change, and we can try it and see the result.
Should we invest time on this?

Otherwise, for the reasons you've mentioned above, I think that's best to exclude Aztec from the config changes code altogether, and introduce another Activity...it could be a big PR though. Wondering if we should following this path for an event (font size change in settings) that is going to happen rarely.

Reply:
> instead we need to reload the pictures/videos and re-layout them on configChanges. It alone is a rather small change, and we can try it and see the result.
> Should we invest time on this?

I think this might be a rabbit hole. Plus, I don't think we know which other code or flow breaks in Aztec if we lift the assumption of the activity re-creation.

OTOH, going the new-Activity way for this particular issue (the font-size change) that's not critical is a bit much. Let's leave this PR unmerged and we give it some more thought?

Reply:
>  going the new-Activity way for this particular issue (the font-size change) that's not critical is a bit much.

Yes, 100% agree with you on the points mentioned above. The only way to be sure to not break Aztec is to duplicate the Activity, but it feels way to much for a very rare writing flow (user needs to change the font size when the editor is open on the screen. If they change the font size and then open the editor the new font size is respected).